{{cookiecutter.project_shortname}}/docker-compose.full.yml

```diff
@@ -30,7 +30,7 @@ services:
     extends:
       file: docker-services.yml
       service: app
-    command: pipenv run uwsgi /opt/invenio/var/instance/uwsgi_ui.ini
+    command: ["uwsgi /opt/invenio/var/instance/uwsgi_ui.ini"]
```

Comment:
Doesn't this just circumvent the pipenv?

Reply:
Yes, @slint suggested to install all dependencies directly to system, not to virtualenv. That's why `pipenv install` has `--sytem` flag.